### ISSUE TITLE
Add poster from image for episodes

### ISSUE THREAD

**Issue by kolbdog3333** [theMK2k/Media-Hoarder#37]
I was just thinking for missing images for episodes when it adds them after it scans for all the metadata you can have an option to choose an image from the imdb photos so the image cover isn't blank or it will use the default image from the tv series poster if an image can't be found so you don't end up with blank episode posters like this 
![Capture](https://github.com/theMK2k/Media-Hoarder/assets/75857608/9ada05f8-f4e0-4712-b6b7-0fad166f2bf1)

**Comment by kolbdog3333** [comment]
And on the imdb website it shows it has an actual image but not a screenshot image here is a picture from the website so you can see what i mean since some episodes have actual images from the episode some don't they have a picture from the tv show poster or a picture that is not a screenshot of teh episdoe here are two so you have idea what i am talking about:
![Capture 2](https://github.com/theMK2k/Media-Hoarder/assets/75857608/c908e729-fe5b-4cfd-9479-da8910a3f891)
![Capture 3](https://github.com/theMK2k/Media-Hoarder/assets/75857608/5e3f7e8f-7345-4099-88ac-531803901abb)

**Comment by theMK2k** [comment]
It's odd that the imdb site provides a poster / screenshot but Media Hoarder wasn't able to pick those up. ("Forged in Fire Christmas" episode) I'll look into that.

I'll also work on a solution if imdb doesn't have an image available.

**Comment by kolbdog3333** [comment]
It did the same thing with season 2 episode 1, Season 2 episode 16 and 17 of Villains of Valley View it seems it only grabs the image if its not a poster for the episode since those episodes dont have actual screenshots of the episode it is same picture as the poster of the show

**Comment by theMK2k** [comment]
Yeah we defo have a pattern here and it's not too bad, because we likely have the series' poster and could re-use it in case the episode does not provide an individual one. At least it would be a start.

**Comment by enzedyo** [comment]
> It's odd that the imdb site provides a poster / screenshot but Media Hoarder wasn't able to pick those up. ("Forged in Fire Christmas" episode) I'll look into that.
> 
> I'll also work on a solution if imdb doesn't have an image available.

Was going to post as a new issue, but saw this one, which is very similar. Have noticed for films, where the movie poster does not meet community guidelines, that we dont get a poster either. Example im referring to is the film "Sexworld" from 1978. 

I tried submitting a "Safe for work" version of the poster, but it got rejected. So for films where there are no options, our only solution may be an option to upload the poster to Media Hoarder, ourselves?

**Comment by theMK2k** [comment]
> I tried submitting a "Safe for work" version of the poster, but it got rejected. So for films where there are no options, our only solution may be an option to upload the poster to Media Hoarder, ourselves?

I see. You can't even find Sexworld 1978 on imdb itself (not logged-in user).

**Comment by kolbdog3333** [comment]
Maybe for the missing episode poster picture if one cant be found it will use the series poster instead.

**Comment by theMK2k** [comment]
> Maybe for the missing episode poster picture if one cant be found it will use the series poster instead.

Good idea. I created a separate ticket for it.
